admin_manual/upgrading/marketplace_apps.rst

```diff
@@ -29,6 +29,6 @@ But here is a concise approach:
 
 .. Links
    
-.. _the tarball: https://download.owncloud.org/community/owncloud-10.0.2.tar.bz2
-.. _the zip archive: https://download.owncloud.org/community/owncloud-10.0.2.zip
+.. _the tarball: https://download.owncloud.org/community/owncloud-10.0.3.tar.bz2
```

Comment:
why only 10.0.3?
(and the line below)

Reply:
This is because it was the latest version at the time of writing.